server/statistics/hot_peer_cache.go

```diff
@@ -142,26 +142,26 @@ func (f *hotPeerCache) CollectMetrics(stats *StoresStats, typ string) {
 		store := storeTag(storeID)
 		threshold := f.calcHotThreshold(stats, storeID)
 		hotCacheStatusGauge.WithLabelValues("total_length", store, typ).Set(float64(peers.Len()))
-		hotCacheStatusGauge.WithLabelValues("hotThreshold", store, typ).Set(float64(threshold))
+		hotCacheStatusGauge.WithLabelValues("hotThreshold", store, typ).Set(threshold)
 	}
 }
 
-func (f *hotPeerCache) getBytesFlow(region *core.RegionInfo) uint64 {
+func (f *hotPeerCache) getBytesFlow(region *core.RegionInfo) float64 {
```

Comment:
IMO, this function shouldn't be changed.

Reply:
Fixed.